scripts/mcptam_install.bash

```diff
@@ -102,7 +102,7 @@ build_mcptam()
 
 	if [ "$CI" = true ]; then
 		# grab pull request version to be tested
-		cd ~
+		cd ..
```

Comment:
In general we should avoid using cd .. as it is making assumptions as to where you are located in the file system, which in turn makes this really hard to debug or follow.

cd ~ is ok since we know it is the same as /home/username and cd. 

Perhaps a variable of the catkin_ws should be used? 
Something like CATKIN_BUILD_PATH="$HOME/catkin_ws" at the top of the script.